src/scripting/game_lua_kernel.cpp

```diff
@@ -2723,7 +2728,8 @@ int game_lua_kernel::intf_set_floating_label(lua_State* L, bool spawn)
 	flabel.set_color(color);
 	flabel.set_bg_color(bgcolor);
 	flabel.set_alignment(alignment);
-	flabel.set_position(x, y);
+	flabel.set_position(x + padding, y + padding);
```

Comment:
This is the main part who behavior I'd like feedback on. By default, floating_label doesn't count its border as part of its position. I think including padding in the bounding box is the more intuitive behavior, at least for the player-facing API.